gridstatus/nyiso.py

```diff
@@ -476,6 +476,33 @@ def _download_nyiso_archive(self, date, end=None, dataset_name=None, filename=No
 
         return df
 
+    def get_capacity_prices(
+        self, date=pd.Timestamp.now(tz="US/Eastern"), verbose=False
+    ):
+        """Pull the most recent capacity market report's market clearing prices
+
+        Parameters:
+            verbose (bool): print out requested url
+
+        Returns:
+            pd.DataFrame: a dataframe of monthly capacity prices (all three auctions) for each of the four capacity localities within NYISO
+        """
+
+        # todo: it looks like the "27447313" component of the base URL changes every year but I'm not sure what the link between that and the year is...
+        capacity_market_base_url = (
+            "https://www.nyiso.com/documents/20142/27447313/ICAP-Market-Report"
+        )
+
+        url = f"{capacity_market_base_url}-{date.month_name()}-{date.year}.xlsx"
+        if verbose:
+            print(url)
+
+        df = pd.read_excel(url, sheet_name="MCP Table", header=[0, 1])
```

Comment:
any value in all adding data from the "UCAP Table" or "Summary Table" sheets? if not now, we can always do it in a future PR if someone needs it

Reply:
Good question, I had a similar thought when making it but decided to stick to just grabbing prices for now since that's the most important/commonly needed data. Unless someone is trying to get super detailed and trying to understand the underlying mechanics of the market and maybe predict the prices would they benefit from looking at that data; in my experience this a lot less common than just grabbing the prices for a historical analysis or something. As you said we can easily add it later if needed.